logging/processors.rst

```diff
@@ -71,7 +71,7 @@ using a processor.
                     type: stream
                     path: '%kernel.logs_dir%/%kernel.environment%.log'
                     level: debug
-                    formatter: app.logger.session_request_processor
+                    formatter: monolog.formatter.session_request
```

Comment:
This change looks correct to me. Can you also update the other configuration formats? They seem to have the same incorrect value at the formatter option.

Reply:
@iltar no problem, I've updated the other configuration formats